### PR TITLE
Pr/commits of week 33

### PR DESCRIPTION
**Pull request by edgargabriel**
it looks like I forgot to file a pr for the split collective shared file pointer tests, so I included it in this pr.

### CODE REVIEW

File: ompi/mca/sharedfp/individual/sharedfp_individual_write.c [hunk 1]
```diff
@@ -104,7 +104,7 @@ int mca_sharedfp_individual_write_ordered (mca_io_ompio_file_t *fh,
 
     if(fh->f_sharedfp_data==NULL){
 	if ( mca_sharedfp_individual_verbose ) {
-	    printf("sharedfp_individual_write - opening the shared file pointer\n");
+	    printf("sharedfp_individual_write_ordered - opening the shared file pointer\n");
```
Comment (hppritcha) [inline ompi/mca/sharedfp/individual/sharedfp_individual_write.c]: may `opal_output` instead of printf?

**Comment by hppritcha** [comment]
except for minor printf/opal_output this looks good to me.

**Comment by edgargabriel** [comment]
I can fix the printf vs. opal_output thing, will add it to this pr a bit later today. Thanks! Also wandering whether I should add the coverty fixes that Jeff sent me over the weekend to the same pr, or should I file a new one?

**Comment by hppritcha** [comment]
As long as the coverity fix goes in as a distinct commit (cherry-picked)
from master that is okay with me if it all goes in as a roll-up PR into
v2.x.  Please don't combine with another commit though - that makes it more
difficult to track.

2015-08-17 8:01 GMT-06:00 Edgar Gabriel notifications@github.com:

**Comment by hppritcha** [comment]
being nice to mlnx bot

retest please

**Comment by edgargabriel** [comment]
@hppritcha I added the commit for cleanup of printf vs. opal_output ( for all three sharedfp components in v2.x). I would suggest that the coverity fix goes in a separate pr. The fix is not entirely trivial and it might be towards the end of the week before I get to it.

**Comment by hppritcha** [comment]
:+1:
@jsquyres  this is ready to merge.

**Pull request merged**